src/main/java/net/kyori/text/serializer/ComponentSerializers.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of text, licensed under the MIT License.
+ *
+ * Copyright (c) 2017 KyoriPowered
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package net.kyori.text.serializer;
+
+import net.kyori.text.Component;
+
+/**
+ * A collection of component serializers.
+ */
+public final class ComponentSerializers {
+
+  public static final ComponentSerializer<Component, String> GSON = GsonComponentSerializer.INSTANCE;
+  /**
+   * @deprecated legacy
+   */
+  @Deprecated
```

Comment:
no need for this to be deprecated

sure the spec underlying the code is deprecated, but this is just going to make peoples ides moan at them, although perhaps it should if they use legac.....

carry on